From ac39ee38d292b713b040cd51cc590470ae9c39ca Mon Sep 17 00:00:00 2001
From: Francis Rowe <info@gluglug.org.uk>
Date: Wed, 5 Aug 2015 02:25:47 +0100
Subject: [PATCH 4/7] mainboard/lenovo/r500: use 4MiB ROM size by default

These systems come with 4MiB chips (SOIC-8), by default, so it
makes more sense to use this size.

Change-Id: I9d8a7dd7b619d7d3af2f725f914d0ee79fcc6d50
Signed-off-by: Francis Rowe <info@gluglug.org.uk>
---
 src/mainboard/lenovo/r500/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mainboard/lenovo/r500/Kconfig b/src/mainboard/lenovo/r500/Kconfig
index d08e5f5..67915f9 100644
--- a/src/mainboard/lenovo/r500/Kconfig
+++ b/src/mainboard/lenovo/r500/Kconfig
@@ -10,7 +10,7 @@ config BOARD_SPECIFIC_OPTIONS # dummy
 	select EC_LENOVO_H8
 	select NO_UART_ON_SUPERIO
 	select DRIVERS_ICS_954309
-	select BOARD_ROMSIZE_KB_8192
+	select BOARD_ROMSIZE_KB_4096
 	select DRIVERS_GENERIC_IOAPIC
 	select HAVE_MP_TABLE
 	select HAVE_ACPI_TABLES
-- 
1.9.1

